src/WebJobs.Script/Config/ScriptHostConfiguration.cs

```diff
@@ -51,5 +51,8 @@ public ScriptHostConfiguration()
         /// specified by <see cref="RootLogPath"/>.
         /// </summary>
         public bool FileLoggingEnabled { get; set; }
+
+        // Resolve appsetting values in the function bindings. 
+        public INameResolver AppSettings { get; set; }
```

Comment:
I'll comment better. This is for resolving the %key% in the function.json files. They'll ultimately resolve to AppSettings (or EnvVars for local debugging; or whatever the local host passes in) 

---
In reply to: [54122965](https://github.com/Azure/azure-webjobs-sdk-script/pull/26#discussion_r54122965) [](ancestors = 54122965)